guides/inventory.txt

```diff
@@ -14,6 +14,11 @@ DESCRIPTION
     project configuration file named 'bolt-project.yaml' alongside the inventory
     file.
 
+    When Bolt loads inventory, it loads the entire inventory, not just the
+    groups and targets specified on the command line. If a target is defined in
+    multiple groups, this might result in target configuration that is different
+    than expected.
```

Comment:
```suggestion
    groups and targets specified on the command line. If you've defined a
    target in multiple groups, this might result in target configuration that
    is different than expected.
```